src/librssguard/services/owncloud/owncloudnetworkfactory.cpp

```diff
@@ -614,7 +614,9 @@ QList<Message> OwnCloudGetMessagesResponse::messages() const {
         enclosure.m_mimeType = QSL("image/png");
       }
 
-      msg.m_enclosures.append(enclosure);
+      if (!message_map[QSL("enclosureMime")].toString().isEmpty() || !enclosure_link.startsWith(QSL("https://www.youtube.com/v/"))) {
```

Comment:
You probably want to use logical AND operator "&&" and not logical OR operator, right?

Reply:
OR is correct. 
If any of the two assumptions is wrong we want to append the enclosure. 
Mime is not empty -> append enclosure.
String is not starting with youtube -> append enclosure.